conf/default.toml

```diff
@@ -122,6 +122,9 @@ background_color = "#fdfbff"
 # Optional; Will limit the possibility to zoom in past a certain zoom level
 # max_zoom = 10
 
+# Optional; Will not tile WMS. False by default.
+# do_not_tile_wms = false
```

Comment:
Maybe it'd make sense to move this param a little further down (just above `do_not_use_default_basemap`) and add some more background info on this here and/or in the doc, like:
`... This can be set `true` to prevent visual conflicts on tile borders, if the WMS server does not add a gutter, for example. gn-ui does not add a gutter on the client side, in order to allow server-side caching.`
What do you think about the formulation @jahow ?

Reply:
> Maybe it'd make sense to move this param a little further down (just above do_not_use_default_basemap)

yes agreed, it should not be above the extent, layers etc.

As for formulation, it can be elaborated a little but the big explanation should be given in the doc website (see https://github.com/geonetwork/geonetwork-ui/pull/887/files#diff-9088b1e53d9bb428254eea7cc21785db455294f9bea3c903806cd8dfa06f7024)